diagnostic.md

```diff
@@ -21,13 +21,34 @@ Write the `curl` request you'd use to retrieve a list of **all donuts** on the
 server.
 
 ```sh
-# your answer here
+
+ curl --include --request GET http://localhost:4741/donuts
 ```
 
 Write an AJAX request to retrieve a list of **all donuts** on the server.
 
 ```js
-let getDonuts = /* your answer here */;
+$(() => {
+  $('#donuts-search').on('submit', donutEvents.getDonuts)
+}
+
+let getDonuts = function(event){
+  event.preventDefault()
+  donutsApi.index()
+  .then(logResponseBody)
+  .catch(logRequestError)
+}
+
+ const index = function(){
```

Comment:
All you need for this diagnostic is this part, adding in the extra stuff probably contributed to you taking longer than expected